.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		"walkme.ruby-extension-pack",
```

Comment:
@PombeirP thoughts on changing this to an explicit list of all extensions used? Seems this includes some template extensions which wouldn't be used.

Reply:
Good point @feelepxyz. I've changed it so that it only includes plugins that are useful to the project.